### PR TITLE
UISINVCOMP-15: ECS - Accept `tenantId` prop to search entries in the specified tenant for supporting `ui-plugin-find-instance` plugin

### PR DESCRIPTION
**Pull request by alisher-epam**
## Purpose

[UISINVCOMP-15](https://folio-org.atlassian.net/browse/UISINVCOMP-15): ECS - Accept `tenantId` prop to search entries in the specified tenant for supporting `ui-plugin-find-instance` plugin
We have a corresponding ticket as well https://github.com/folio-org/ui-plugin-find-instance/pull/235
#### TODOS and Open Questions

https://github.com/user-attachments/assets/90f192ba-2469-4997-8263-ac26c2989c30

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [ ] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: lib/hooks/useCommonData/useCommonData.js [hunk 1]
```diff
@@ -20,22 +20,22 @@ import {
...
+  const { materialTypes, isLoading: isMaterialTypesLoading } = useMaterialTypesQuery(tenantId);
+  const { natureOfContentTerms, isLoading: isNatureOfContentTermsLoading } = useNatureOfContentTermsQuery(tenantId);
+  const { holdingsTypes, isLoading: isHoldingsTypesLoading } = useHoldingsTypesQuery(tenantId);
+  const { statisticalCodeTypes, isLoading: isStatisticalCodeTypesLoading } = useStatisticalCodeTypesQuery(tenantId);
+  const { statisticalCodes, isLoading: isStatisticalCodesLoading } = useStatisticalCodesQuery(tenantId);
+  const { modesOfIssuance, isLoading: isModesOfIssuanceLoading } = useModesOfIssuanceQuery(tenantId);
+  const { instanceStatuses, isLoading: isInstanceStatusesLoading } = useInstanceStatusesQuery(tenantId);
+  const { instanceFormats, isLoading: isInstanceFormatsLoading } = useInstanceFormatsQuery(tenantId);
+  const { holdingsSources, isLoading: isHoldingsSourcesLoading } = useHoldingsSourcesQuery(tenantId);
+  const { instanceTypes, isLoading: isInstanceTypesLoading } = useInstanceTypesQuery(tenantId);
```
Comment (usavkov-epam) [inline lib/hooks/useCommonData/useCommonData.js:38]: I'd suggest keeping the current pattern to provide an options object in this case

Comment (alisher-epam) [reply lib/hooks/useCommonData/useCommonData.js]: updated it

Comment (alisher-epam) [reply lib/hooks/useCommonData/useCommonData.js]: https://github.com/user-attachments/assets/7c028772-c4f1-4b88-8936-9eef36100188

**Review by Dmytro-Melnyshyn — APPROVED** [review]

**Comment by BogdanDenis** [comment]
Hey, maybe I'm missing something- but could you explain why this is needed?
By default `useOkapiKy` uses as tenant whatever ECS tenant user is currently on, so when they switch to a member tenant the requests will automatically include that tenant id.
Here's existing behaviour on https://folio-testing-sprint-cs00000int.ci.folio.org/:

https://github.com/user-attachments/assets/fdddc1f5-5e03-494c-8804-b320f48f9ced

**Comment by MikeTaylor** [comment]
> Hey, maybe I'm missing something - but could you explain why this is needed?

I wondered that, too.

**Comment by alisher-epam** [comment]
@BogdanDenis Basically, we have a new "Central ordering" feature that has been introduced where a central tenant can perform some action on behalf of the member tenants. In order to perform such actions we are displaying list of tenants for central tenant only and central tenant can select locations from member tenants. Here is the ticket for the plugin https://folio-org.atlassian.net/browse/UIPFI-157

**Review by BogdanDenis — APPROVED** [review]

**Pull request merged**